hdfs_setup.sh

```diff
@@ -2,8 +2,8 @@
 
 DSOURCES=('flow' 'dns')
 DFOLDERS=('binary' 'csv' 'hive' 'stage')
-DBNAME='duxbury'
-source /etc/duxbay.conf
+DBNAME='oni'
+source /etc/oni.conf
 
```

Comment:
I think we should add "export DBNAME" here and use the oni.conf file to set this as it's taking the variable from line 5 instead.

the same could be said for DSOURCES and DFOLDERS